functions.py

```diff
@@ -11,13 +11,14 @@
 
 
 async def handle_response(message: disnake.Message):
+    await message.channel.trigger_typing()
```

Comment:
This will start typing before we check if it was something we can respond to or not. So it will keep going for the entire timeout duration and stop and the user will just wonder where the response is